src/validation.cpp

```diff
@@ -1058,21 +1058,6 @@ bool MemPoolAccept::ReplacementChecks(Workspace& ws)
     TxValidationState& state = ws.m_state;
 
     CFeeRate newFeeRate(ws.m_modified_fees, ws.m_vsize);
```

Comment:
In "Implement new RBF logic for cluster mempool" e6315c24326016cfaee5bd046e8b2e4e1088ac6b

IMO think there should be a BIP or a document in doc accompanying this that explicitly highlight the new RBF rules for cluster mempool, because we reference the number of bip125 rules and they not exactly like it like rule 5 was changed instead of transactions conflicts its now  distinct cluster conflicts.